pyproject.toml

```diff
@@ -0,0 +1,81 @@
+[tool.poetry]
+name = "reactivex"
+version = "4.0.0a1"
+description = "Reactive Extensions (Rx) for Python"
+readme = "README.rst"
+authors = ["Dag Brattli <dag@brattli.net>"]
+license = "MIT License"
+homepage = "http://reactivex.io"
+repository = "https://github.com/ReactiveX/RxPY"
+documentation = "https://rxpy.readthedocs.io/en/latest/"
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Environment :: Other Environment",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: Implementation :: CPython",
+    "Topic :: Software Development :: Libraries :: Python Modules",
+]
+packages = [
+    { include = "rx" },
+    { include = "rx/py.typed" }
+]
+
+[tool.poetry.dependencies]
+python = ">= 3.7, < 3.11"
```

Comment:
Does this mean a release with this will not install on python > 3.10?
If yes, this may be too conservative.

Reply:
It says that it needs that be less than 3.11. That includes 3.10, but does not include 3.11 (which is in alpha-5)

Reply:
We can enable it if someone tries 3.11 and confirms it works with